src/chrome/MiniButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import Icon, { IconSize } from './Icon'
+
+import { ButtonType, generateButtonTypeClasses } from './Button'
+
+export interface MiniButtonProps {
```

Comment:
what about calling this `IconOnlyButton`, since it's a button that can't have text? Feels strange to have a `MiniButton` with a `size='lg'` prop